core/src/test/java/tech/tablesaw/filters/TimeDependentFilteringTest.java

```diff
@@ -113,11 +114,11 @@ public static void main(String[] args) throws Exception {
       List<LocalDate> eventDates = new ArrayList<>();
 
       // iterate an individual table and find the rows where concept matches the target concept
-      for (int row : patientTable) {
+      for (Row row : patientTable) {
         StringColumn concepts = patientTable.stringColumn("concept");
         DateColumn dates = patientTable.dateColumn("date");
-        if (concepts.get(row).equals(conceptZ)) {
-          eventDates.add(dates.get(row));
+        if(row.getString("concept").equals(conceptZ)) {
```

Comment:
`concepts` and `dates` are now unused. Should they be removed?

Reply:
I am not sure why this line was touched in this PR. I think it was my first PR. If something related to concepts and dates needs to be cleaned up can you create a separate issue?